app/search/routes.py

```diff
@@ -454,6 +456,7 @@ def get_dataset_metadata_information(dataset):
         "licenses": datsdataset.licenses,
         "sources": datsdataset.sources,
         "parentDatasets": datsdataset.parentDatasetId,
+        "primaryPublications": datsdataset.primaryPublications,
```

Comment:
Note: this one is definitely needed to display the publication on the dataset page